plugins/uefi/fu-self-test.c

```diff
@@ -13,8 +13,55 @@
 #include "fu-uefi-bgrt.h"
 #include "fu-uefi-common.h"
 #include "fu-uefi-device.h"
+#include "fu-uefi-pcrs.h"
 #include "fu-uefi-vars.h"
 
+static void
+fu_uefi_pcrs_1_2_func (void)
+{
+	gboolean ret;
+	g_autoptr(FuUefiPcrs) pcrs = fu_uefi_pcrs_new ();
+	g_autoptr(GError) error = NULL;
+	g_autoptr(GPtrArray) pcr0s = NULL;
+	g_autoptr(GPtrArray) pcrXs = NULL;
+
+	ret = fu_uefi_pcrs_setup (pcrs, &error);
+	g_assert_no_error (error);
+	g_assert_true (ret);
+	pcr0s = fu_uefi_pcrs_get_checksums (pcrs, 0);
+	g_assert_nonnull (pcr0s);
+	g_assert_cmpint (pcr0s->len, ==, 1);
+	pcrXs = fu_uefi_pcrs_get_checksums (pcrs, 999);
+	g_assert_nonnull (pcrXs);
+	g_assert_cmpint (pcrXs->len, ==, 0);
+}
+
+static void
+fu_uefi_pcrs_2_0_func (void)
+{
+	gboolean ret;
+	g_autoptr(FuUefiPcrs) pcrs = fu_uefi_pcrs_new ();
+	g_autoptr(GError) error = NULL;
+	g_autoptr(GPtrArray) pcr0s = NULL;
+	g_autoptr(GPtrArray) pcrXs = NULL;
+
+	g_setenv ("FWUPD_UEFI_TPM2_YAML_DATA",
+		  "sha1 :\n"
+		  "  0  : cbd9e4112727bc75761001abcb2dddd87a66caf5\n"
+		  "sha256 :\n"
+		  "  0  : 122de8b579cce17b0703ca9f9716d6f99125af9569e7303f51ea7f85d317f01e\n", TRUE);
+
```

Comment:
I think you should have a second test of a failure scenario (where you get bad output). While I was setting up the tools locally I know I definitely saw some output that was just a line or two with an error.